doc/webui/index.rst

```diff
@@ -0,0 +1,124 @@
+respective sections in the documentation.
+
+.. note:: For productive environments, it is absolutely necessary to secure the connection with HTTPS.
+   PrivacyIDEA only comes with a self-signed certificate, which serves for testing environments.
+
+Tokens
+-------
+
+.. index:: tokensview
+
+The administrator can see all the tokens of all realms he is allowed to manage in the
+tokenview. Each token can be located in several realms and be assigned to one
+user. The administrator can see all the details of the token.
+
+.. figure:: images/tokensview.png
+   :width: 500
+
+   *Tokens overview*
+
+The administrator can click on one token, to show more details of this token
+and to perform actions on this token. Read on in :ref:`token_details`.
+
+
+Users
+-------
+
+The administrator can see all users fetched by :ref:`useridresolvers` located in
+:ref:`realms` he is allowed to manage.
+
+.. note:: Users are only visible, if the useridresolver is located
+   within a realm. If you only define a useridresolver but no realm,
+   you will not be able to see the users!
+
+You can select one of the realms in the left drop down box. The administrator
+will only see the realms in the drop down box, that he is allowed to manage.
+
+
+.. figure:: images/usersview.png
+   :width: 500
+
+   *The Users view list all users in a realm.*
+
+The list shows the users from the select realm. The username, surname,
+given name, email and phone are filled according to the definition of
+the useridresolver.
+
+Even if a realm contains several useridresolvers all users from all
+resolvers within this realm are displayed.
+
+Read about the functionality of the users view in the following sections.
+
+.. toctree::
+   :maxdepth: 1
+
+   user_details
+   manage_users
+
+
+Machines
+-------
+
+.. todo:: Short sentence about the purpose of machine-resolving.
+
+The :ref:`machines` which are...
+
+.. figure:: images/machinesview.png
+   :width: 500
+
+    *The Machines view.*
+
+
+
+Config
+-------
+
+The configuration tab is the heart of the privacyIDEA server. It contains the general
+:ref:`system_config`, allows configuring :ref:`policies` which are important to configure
+behavior of the system, manages the :ref:`eventhandler` and lets the user set up :ref:`periodic_tasks`.
+
+.. figure:: ../configuration/images/system-config.png
+   :width: 500
+
+    *The Config section is the heart of the privacyIDEA server.*
+
+Audit
+-------
+
+In this tab, the :ref:`audit` log is displayed which lists all events the server registers.
+
+.. figure:: ../audit/auditlog.png
+   :width: 500
+
+    *Events can be displayed in the Audit log.*
+
+.. _components:
+
+Components
+-------
+
+.. index:: Components
+
+Starting with privacyIDEA 2.15 you can see privacyIDEA components in the Web UI.
+privacyIDEA collects authenticating clients with their User Agent. Usually
+this is a type like *PAM*, *FreeRADIUS*, *OTRS*, *Wordpress*...
```

Comment:
We could remove OTRS and add *ownCloud* and *privacyIDEA Credential Provider*.

Reply:
Done. Also added a link to the plugins.

Reply:
done. I linkes to the plugins  section.